### PR TITLE
Fix VerifySignature command json marshaling

### PR DESCRIPTION
**Pull request by rgnote**
Temporary fix for https://github.com/notaryproject/notation-core-go/issues/88

Signed-off-by: rgnote <5878554+rgnote@users.noreply.github.com>

### CODE REVIEW

File: verification/verifier_helpers.go [hunk 1]
```diff
@@ -253,12 +253,12 @@ func (v *Verifier) executePlugin(ctx context.Context, trustPolicy *TrustPolicy,
...
 	}
-	var attributesToProcess []interface{}
-	extendedAttributes := make(map[interface{}]interface{})
+	var attributesToProcess []string
+	extendedAttributes := make(map[string]interface{})
 
 	for _, attr := range getNonPluginExtendedCriticalAttributes(signerInfo) {
-		extendedAttributes[attr.Key] = attr.Value
-		attributesToProcess = append(attributesToProcess, attr.Key)
+		extendedAttributes[attr.Key.(string)] = attr.Value
```
Comment (patrickzheng200) [inline verification/verifier_helpers.go:260]: This line of code would panic if attr.Key is not of type string. Type assertion is needed here.

File: internal/mock/mocks.go [hunk 2]
```diff
@@ -3,7 +3,6 @@ package mock
 import (
 	"context"
 	_ "embed"
-
```
Comment (shizhMSFT) [inline internal/mock/mocks.go:6]: It is better to keep those import block seperated.

File: verification/verifier_helpers.go (hunk 1 above)
Comment (rgnote) [reply verification/verifier_helpers.go:260]: This is handled in `getNonPluginExtendedCriticalAttributes`. It is returning only the string attributes

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**